tutorials/tour/basics.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Variables
+
+You can define variables with `var` keyword.
+
+```
+var x = 1 + 1
+x += 1
+```
+
+Often, you want your variables to be immutable. You can use `val` keyword in that case.
```

Comment:
@a1russell, I don't necessarily agree with this sentiment. Because you can write code that is essentially Java with a `.scala` extension and pass it through the Scala compiler, I think it's extremely important to focus on the basics of what Scala really is in this article. When it comes to immutable vs mutable references (`val` vs `var), Scala is all about `val`s. It's rare to have to use a `var` anywhere in a normal Scala application. Therefore, I think introducing `val`s should probably come first, and `var`s should probably be introduced as that thing from Java that you can use if you absolutely have to, but is generally avoided.